src/main/java/org/antvoice/beam/PubsubMessageProcessor.java

```diff
@@ -50,7 +50,7 @@ public void processElement(ProcessContext c) {
 
             String dataset = metadata.get("dataset");
             String table = metadata.get("table");
-            _counterProvider.getCounter("PubsubMessageProcessor", dataset + "." + table).inc();

```

Comment:
Maybe we should keep both counters, they are both interesting. Unless you are afraid we'll hit the google cloud limit